src/app/shared/register-github-app/register-github-app.component.html

```diff
@@ -0,0 +1,38 @@
+<div>
+  <p>
+    Install the Dockstore GitHub app on your repositories. This gives GitHub permission to notify Dockstore when pushes are made to GitHub.
+    For more details, see our
+    <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps/github-apps.html" target="_blank" rel="noopener noreferrer">
+      documentation <mat-icon>open_in_new</mat-icon> </a
+    >.
+  </p>
+  <p>Perform the following steps in order:</p>
+  <ol>
+    <li>Click the button below to install the Dockstore GitHub app on your GitHub organizations or specific repositories.</li>
+    <li>
+      Push a valid .dockstore.yml to the root of a GitHub repo that has the Dockstore GitHub app installed. The .dockstore.yml tells
+      Dockstore where the {{ entryType }}'s files are located.
```

Comment:
This might be pushing it given how much room we have, but maybe we could link the documentation for the [.dockstore.yml templates](https://docs.dockstore.org/en/stable/assets/templates/template.html)?